### PR TITLE
Update Get-DtcAdvancedHostSetting.md

### PR DESCRIPTION
**Pull request by sowjanya365**
Issue - "fill in the blank" assignments - Description has been updated now.

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws306 @tseward

Peer review needed for this PR. Thanks.

**Comment by tseward** [comment]
Agree with @dariomws306, otherwise looks good.

File: docset/winserver2012r2-ps/msdtc/Get-DtcAdvancedHostSetting.md [hunk 1]
```diff
@@ -47,7 +47,14 @@ This command queries the tracing setting for TRACE_CM.
 ## PARAMETERS
 
 ### -AsJob
-{{Fill AsJob Description}}
+Runs the cmdlet as a background job. Use this parameter to run commands that take a long time to complete. 
+
+The cmdlet immediately returns an object that represents the job and then displays the command prompt. 
+You can continue to work in the session while the job completes. 
+To manage the job, use the `*-Job` cmdlets. 
+To get the job results, use the [Receive-Job](http://go.microsoft.com/fwlink/?LinkID=113372) cmdlet.
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/msdtc/Get-DtcAdvancedHostSetting.md]:
```suggestion
To get the job results, use the [Receive-Job](https://go.microsoft.com/fwlink/?LinkID=113372) cmdlet. 
```

File: docset/winserver2012r2-ps/msdtc/Get-DtcAdvancedHostSetting.md [hunk 2]
```diff
@@ -47,7 +47,14 @@ This command queries the tracing setting for TRACE_CM.
...
 ### -AsJob
-{{Fill AsJob Description}}
+Runs the cmdlet as a background job. Use this parameter to run commands that take a long time to complete. 
+
+The cmdlet immediately returns an object that represents the job and then displays the command prompt. 
+You can continue to work in the session while the job completes. 
+To manage the job, use the `*-Job` cmdlets. 
+To get the job results, use the [Receive-Job](http://go.microsoft.com/fwlink/?LinkID=113372) cmdlet. 
+
+For more information about Windows PowerShell background jobs, see [about_Jobs](http://go.microsoft.com/fwlink/?LinkID=113251).
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/msdtc/Get-DtcAdvancedHostSetting.md]:
```suggestion
For more information about Windows PowerShell background jobs, see [about_Jobs](https://go.microsoft.com/fwlink/?LinkID=113251).
```

**Comment by sowjanya365** [comment]
> Please use https instead of http. Thanks!

Hi @dariomws306 

Thanks for the inputs. Updated now. Thank you!

**Pull request merged**